app/src/main/java/com/example/calculator/Calculate.kt

```diff
@@ -0,0 +1,97 @@
+package com.example.calculator
+
+import java.util.ArrayDeque
+
+class Calculate {
+    private var list : MutableList<Any> = mutableListOf()
+
+    fun digitsOperator(txtInput : String) {
+        var currentDigit = ""
+        for (character in txtInput) {
+            if (character.isDigit() || character == '.') {
+                currentDigit += character
+            } else  {
+                list.add(currentDigit.toFloat())
+                list.add(character)
+                currentDigit = ""
+            }
+        }
+        if (currentDigit != "") {
+            list.add(currentDigit.toFloat())
+        }
+    }
+
+    fun clearList() {
+        list.clear()
+    }
+
+    fun result() : Float {
+        return evaluatePostfix()
+    }
+
+
+    // kiem tra do lon cua dau cong tru nhan chia
+    private fun priority(operator : Any) : Int {
+        if (operator !is Char) return -2
+        when (operator) {
+            '+', '-' -> return 0
+            '×', '÷' -> return 1
+        }
+        return -1
+    }
+
+    //tinh gia tri postfix
+    private fun evaluatePostfix() : Float {
+        val postFix = infixToPostfix()
+        val stack = ArrayDeque<Any>()
+            for (i in postFix) {
+                if (i is Float) {
+                    stack.push(i)
+                } else {
+                    val x = stack.pop() as Float
+                    var y = stack.pop() as Float
+                    when (i) {
+                        '+' -> y += x
+                        '-' -> y -= x
+                        '×' -> y *= x
+                        '÷' -> y /= x
+                    }
+                    stack.push(y)
+                }
+            }
+        return stack.pop() as Float
+    }
+
+    private fun infixToPostfix() : MutableList<Any> {
```

Comment:
đặt tên hàm bắt đầu bằng động từ 
```suggestion
    private fun convertInfixToPostfix() : MutableList<Any> {
```